README.md

```diff
@@ -1 +1,89 @@
-# lodge
\ No newline at end of file
+# lodge
+
+A simple log lib used by Neoway's ML Platform team.
+
+## Features
+
+* Easy to use
+* Env var configurations
+* Structured production log ready!
+* Options to customize fields
+* Set log level by module
+* Based on stdlib
+
+## Getting Started
+
+```python
```

Comment:
@mfrata just leaving a suggestion for formatting your examples. I think use comments can cause confusion. My suggestion includes use `$` for shell commands, `>>>` for python commands, and nothing for outputs.

```python
$ export LOG_LEVEL=ERROR

>>> from lodge import log
>>> log.info("This will not appear")
>>> log.error("Oh no")
{"message": "Oh no", "timestamp": "2021-03-21 14:26:54,839", "level": "ERROR"}
```

What do u think about?

Reply:
I like your suggestion, @brenocosta0901. I agree with it, make it clear.

Reply:
Gonna add it!